test/router_device_channels_worker_SUITE.erl

```diff
@@ -562,20 +600,12 @@ late_packet_test(Config) ->
                 <<"lat">> => <<"unknown">>,
                 <<"long">> => <<"unknown">>
             }
-        ],
-        <<"channels">> => [
-            #{
-                <<"id">> => ?CONSOLE_HTTP_CHANNEL_ID,
-                <<"name">> => ?CONSOLE_HTTP_CHANNEL_NAME,
-                <<"reported_at">> => fun erlang:is_integer/1,
-                <<"status">> => <<"success">>,
-                <<"description">> => '_'
-            }
-        ]
+        }
     }),
 
     %% Make sure we did not get a duplicate with that late message
     receive
+        %% REVIEW: Wrong shape, what messages do we actually want to break the test?
```

Comment:
?

Reply:
get rid of this `receive` block. no longer makes sense in this test.